docs/docs-beta/docs/dagster-plus/features/ci-cd/branch-deployments/dagster-cloud-cli.md

```diff
@@ -4,4 +4,155 @@ sidebar_position: 300
 unlisted: true
 ---
 
-{/* TODO copy from https://docs.dagster.io/dagster-plus/managing-deployments/dagster-plus-cli#using-the-dagster-cloud-cli */}
\ No newline at end of file
+:::note
```

Comment:
I am not sure this is the correct document to migrate. It is a full overview of the `dagster-cloud` CLI and not specific to branch deployments:

- https://docs.dagster.io/dagster-plus/managing-deployments/dagster-plus-cli#using-the-dagster-cloud-cli

Reply:
Yeah, I'm not sure why I put that in there, that doesn't make sense. Feel free to delete it!